sdk/state/payment.go

```diff
@@ -111,6 +119,10 @@ func (c *Channel) validatePayment(ca CloseAgreement) (err error) {
 		ca.Details.ConfirmingSigner.Address() != c.remoteSigner.Address() {
 		return fmt.Errorf("close agreement confirmer does not match a local or remote signer, got: %s", ca.Details.ConfirmingSigner.Address())
 	}
+	if !c.latestAuthorizedCloseAgreement.isEmpty() && c.latestAuthorizedCloseAgreement.Details.ObservationPeriodTime == 0 &&
```

Comment:
I was considering adding a helper method like `isAuthorized()` instead for this check, but thought using `!isEmpty()` is good enough

Reply:
❗ Same comment as above, we need to check the unauthorized as well.